src/shared_modules/dbsync/tests/sqlite/sqlite_test.cpp

```diff
@@ -237,3 +237,41 @@ TEST_F(SQLiteTest, ColumnValue)
     EXPECT_DOUBLE_EQ(4.0, spColumn5->value(double_t{}));
     EXPECT_EQ(SQLITE_FLOAT, spColumn5->type());
 }
+
+TEST_F(SQLiteTest, StatementExpand)
+{
+    std::shared_ptr<IConnection> spConnection{ new Connection };
+    Statement createStmt
+    {
+        spConnection,
+        "CREATE TABLE test_table (Colum1 INTEGER, Colum2 TEXT, Colum3 BIGINT, Colum4 BIGINT, Colum5 FLOAT);"
+    };
+    createStmt.step();
+    Statement selectStmt
+    {
+        spConnection,
+        R"(INSERT INTO test_table (Colum1, Colum2, Colum3, Colum4, Colum5) VALUES (?,?,?,?,?);)"
+    };
+    const auto expectedStringStmt{ selectStmt.expand() };
+    EXPECT_TRUE(expectedStringStmt.compare("INSERT INTO test_table (Colum1, Colum2, Colum3, Colum4, Colum5) VALUES (NULL,NULL,NULL,NULL,NULL);") == 0);
```

Comment:
EXPECT_EQ() could be used here, I think it would improve readability.

Reply:
Agreed with @jftuduri